<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventTimeEditWidget</class>
 <widget class="QWidget" name="EventTimeEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>117</width>
    <height>134</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QSpinBox" name="hourSpin">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>h</string>
        </property>
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minuteSpin">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>m</string>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="secondSpin">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="daySpin">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>d</string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="directionCombo">
        <item>
         <property name="text">
          <string>After</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Before</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
